style.css

```diff
@@ -0,0 +1,59 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Inter', sans-serif;
+}
+
+body {
+  padding: 24px;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.section-1 {
+  background-image: url(img/header-illsutration-mobile.svg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+  width: 100%;
+  height: 100vh;
+}
+
+.heading-1 {
+  font-style: normal;
+  font-weight: bold;
+  font-size: 30px;
+  text-align: center;
+  margin: 0 24px;
+  margin-top: 25px;
+  color: #172b4d;
+}
+
+.heading-2 {
+  color: #36b37e;
+  text-align: center;
+  font-size: 30px;
+}
+
+.par-1 {
+  font-size: 20px;
+  text-align: center;
+  color: #172b4d;
+  margin-top: 35px;
+}
+
+.icons {
+  list-style: none;
+  display: flex;
+  justify-content: space-around;
+  margin: 0 50px;
+  margin-top: 30px;
+}
```

Comment:
- [ ] Kindly make the icons the same size as that of Figma's template. You could make the size of the icon either `20px` or `1.25rem`